pyroute2/netlink/nfnetlink/nftsocket.py

```diff
@@ -46,6 +47,55 @@
 NFT_MSG_GETFLOWTABLE = 23
 NFT_MSG_DELFLOWTABLE = 24
 
+# from nftables/include/datatype.h
```

Comment:
Just a comment, what do you think — would we benefit of using `enum` in these cases instead of a plain definition of variables?

Not starting with this PR, but in some future maybe.

Reply:
Yes I think it would be better